setup.md

```diff
@@ -0,0 +1,234 @@
+
+## Add a custom package
+
+Custom package definitions in your composer.json can be added via "Add Package" -> "Custom Package". In the textarea you are then able to paste the entire package definition and select a credential if one is necessary to access the zip files.
+
+Example:
+```json
+{
+    "type": "package",
+    "package": {
+        "name": "smarty/smarty",
+        "version": "3.1.7",
+        "dist": {
+            "url": "https://www.smarty.net/files/Smarty-3.1.7.zip",
+            "type": "zip"
+        },
+        "source": {
+            "url": "http://smarty-php.googlecode.com/svn/",
+            "type": "svn",
+            "reference": "tags/Smarty_3_1_7/distribution/"
+        },
+        "autoload": {
+            "classmap": ["libs/"]
+        }
+    }
+}
+```
+
+The "package" key in a package repository may be set to an array to define multiple versions of a package:
+
+```json
+{
+    "type": "package",
+    "package": [
+        {
+            "name": "foo/bar",
+            "version": "1.0.0",
+            ...
+        },
+        {
+            "name": "foo/bar",
+            "version": "2.0.0",
+            ...
+        }
+    ]
+}
+```
+Once a custom package has been added to Private Packagist it will also benefit from us mirroring the zip file and provide additional endpoint to download the file in case the original storage becomes unavailable.
+
+## Add your Satis instance
+
+Once you start using Private Packagist it usually also replaces your previous Satis set up. Via "Add Package" -> "Json Import" you can paste your satis.json into the textarea and select which credentials should be used to try to import the packages. This will add all packages from your Satis instant to Private Packagist and will make Satis obsolete and therefore one less thing you have to worry about.
+
+## Add a mirrored third party repositories
+On your organisation’s settings page under “Manage Mirrored Repositories” you can add additional third party mirrored repositories. By default packagist.org is enabled for all organisations so there is no need to set that up.
+
+Similar to how Private Packagist can access private VCS repositories it can also access private mirrored third party repositories e.g. repo.magento.com. First create the credentials that are necessary e.g. your Magento Marketplace access keys and then add the mirrored third party repository. Make sure to select the credential while creating the mirror. All packages added from the repository will now automatically use that credential.
```

Comment:
Should we mention to run composer update mirrors again to add all the packages from the new mirror?